assets/source/components/prelaunch-notice/index.js

```diff
@@ -25,9 +36,15 @@ const PrelaunchNotice = () => {
 			</p>
 			<p>
 				<a
-					href="https://help.pinterest.com/en-gb/business/article/get-a-business-profile"
-					target="_blank"
-					rel="noreferrer"
+					{ ...documentationLinkProps( {
+						href:
+							wcSettings.pinterest_for_woocommerce.pinterestLinks
+								.preLaunchNotice,
+						eventName: 'get_started_notice_link_click',
+						linkId: 'prelaunch-notice',
+						context: 'pinterest-landing', //todo: Use helper function to get context from path see https://github.com/woocommerce/pinterest-for-woocommerce/pull/295
+						rel: 'noreferrer',
+					} ) }
```

Comment:
As mentiones, we re-use here `documentationLinkProps ` adding `eventName` as a param